ResearchKit/Localized/en.lproj/Localizable.strings

```diff
@@ -129,6 +129,14 @@
 "TASK_COMPLETE_TITLE" = "Activity Complete";
 "TASK_COMPLETE_TEXT" = "Your data will be analyzed and you will be notified when your results are ready.";
 
+/* Image catpure step. */
+"CAPTURE_BUTTON_CAPTURE_IMAGE" = "Capture Image";
+"CAPTURE_BUTTON_RECAPTURE_IMAGE" = "Recapture Image";
+"CAPTURE_ERROR_INPUT_NOT_FOUND" = "This device does not contain a camera.  This step cannot be completed.";
+"CAPTURE_ERROR_OUTPUT_NOT_FOUND" = "This device cannot capture an image.  In order to complete this step, allow this app access to the camera in the settings.";
```

Comment:
I only use CAPTURE_ERROR_OUTPUT_NOT_FOUND when this if statement returns false:
```
    AVCaptureStillImageOutput *stillImageOutput = [[AVCaptureStillImageOutput alloc] init];
    if ([_captureSession canAddOutput:stillImageOutput])
```
but the only reason that canAddOutput would be returning false is indeed likely to be permissions being denied.  I'll remove the first sentence.